cmake/modules/ProjectVersioning/version.h.in

```diff
@@ -0,0 +1,35 @@
+#ifndef @CMAKE_PROJECT_NAME@_VERSION_H
+#define @CMAKE_PROJECT_NAME@_VERSION_H
+
+
+#include <stdint.h>
+
+#include "@i_export_header@"
+
+
+#ifdef __cplusplus
+extern "C" {
```

Comment:
The whole project is in C++ why do you need the __cplusplus check and C-style name mangling?

Reply:
Sorry, once again leftovers of ancient ages. I shall remove it.